packages/sanity/src/core/studio/StudioLayout.tsx

```diff
@@ -113,13 +115,35 @@ export function StudioLayout() {
 
   const Navbar = useNavbarComponent()
 
+  /**
+   * Handle legacy URL redirects from `/desk` to `/structure`
+   */
+  const isLegacyDeskRedirect =
```

Comment:
I think this makes sense. I think the only corner case I can think of is if a customer had a custom tool named 'structure' and so they explicitly set the deskTool name to 'desk', this will cause an unwanted redirect, but maybe that is not likely.

Reply:
Good point! I just tested, but it seems to work as expected 🤔

Reply:
Does the redirect kick in if someone purposefully names their structure tool `desk` because they want `/desk` to be the path?

Reply:
It doesn't! You can explicity set `name: 'desk', title: 'Desk'`to get the old behavior back.